tutorial/5. hypercube.ipynb

```diff
@@ -0,0 +1,931 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "sns.lineplot(data=df_expanded, x='time.s', y='CONC.A.mol m-3', errorbar=('ci', 95), err_kws={'alpha' : 0.4}, label='CONC.A.mol m-3')\n",
+    "sns.lineplot(data=df_expanded, x='time.s', y='CONC.B.mol m-3', errorbar=('ci', 95), err_kws={'alpha' : 0.4}, label='CONC.B.mol m-3')\n",
+    "sns.lineplot(data=df_expanded, x='time.s', y='CONC.C.mol m-3', errorbar=('ci', 95), err_kws={'alpha' : 0.4}, label='CONC.C.mol m-3')\n",
+    "plt.title('Average concentration with CI over time')\n",
+    "plt.ylabel('Concentration (mol m-3)')\n",
+    "plt.xlabel('Time (s)')\n",
+    "plt.legend(loc='center right')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7758e071",
+   "metadata": {},
+   "source": [
+    "## 6b. Visualizing the Results (Min-Max Curve)\n",
+    "This figure is similar to the CI curve, but it instead displays the range of values for each species at each time step.</br>\n",
```

Comment:
The range of max/min is quite large compared to the CI plot. Does it mean that the max/min sampling data can be significantly different from the 95% data points?